usrp3/tools/scripts/uhd_image_builder.py

```diff
@@ -327,12 +374,35 @@ def get_scriptpath():
     """
     return os.path.dirname(os.path.realpath(__file__))
 
+def get_basedir():
+    """
+    returns the base directory (BASE_DIR) used in rfnoc build process
+    """
+    return os.path.abspath(os.path.join(get_scriptpath(), '..', '..', 'top'))
+
+def get_relative_path(base, target):
+    """
+    Find the relative path (including going "up" directories) from base to target
+    """
+    basedir = os.path.abspath(base)
+    prefix = os.path.commonprefix([basedir, os.path.abspath(target)])
+    path_tail = os.path.relpath(os.path.abspath(target), prefix)
+    total_path = path_tail
+    if prefix != "":
+        while basedir != os.path.abspath(prefix):
+            basedir = os.path.dirname(basedir)
+            total_path = os.path.join('..', total_path)
+        return total_path
+    else:
+        print ("Could not determine relative path")
+        return path_tail
+
 def main():
     " Go, go, go! "
     args = setup_parser().parse_args()
     vfile = create_vfiles(args)
     file_generator(args, vfile)
-    append_item_into_file(args.device, args.include_dir)
+    create_oot_include(args.device, args.include_dir)
```

Comment:
A good question-- the concern would be if the Makefile.OOT.inc generates some syntax error that crashes the FPGA make process. But, I just tested without a -I field, and the generated Makefile.OOT.inc file is empty, which will build the FPGA correctly.